### PR TITLE
tests: improve coverage with integration tests

### PR DESCRIPTION
**Pull request by blueyed**
- install luacov.runner in tests/_runner.lua.
- use Lua's `dofile` to execute the test files, which will give us
  coverage for them.

### CODE REVIEW

**Comment by blueyed** [comment]
Yay, tests are included in coverage, which allows to see how test-menubar is not being executed: https://codecov.io/gh/awesomeWM/awesome/src/45c2b22c9c8348e86705548b88a5146f68b096bb/tests/test-menubar.lua.

**Comment by blueyed** [comment]
You can browse the files from here: https://codecov.io/gh/awesomeWM/awesome/tree/45c2b22c9c8348e86705548b88a5146f68b096bb/tests

**Comment by blueyed** [comment]
Going to merge this soon.

File: .travis.yml [hunk 1]
```diff
@@ -213,8 +213,8 @@ script:
       travis_run_in_fold "make.check-qa" make check-qa
     fi
   - |
-    set -e
     if [ "$TEST_PREV_COMMITS" = 1 ] && ! [ "$TRAVIS_PULL_REQUEST" = false ]; then
+      set -e
```
Comment (psychon) [inline .travis.yml]: The last commit doesn't do what the commit message says and the ones before sound like they should be squashed.

Comment (blueyed) [reply .travis.yml]: Sure.. :)

**Pull request merged**